controller/src/main/java/io/pravega/controller/store/InMemoryScope.java

```diff
@@ -57,7 +57,6 @@ public String getName() {
     public CompletableFuture<Void> createScope() {
         this.sortedStreamsInScope = new TreeMap<>(Integer::compare);
         this.streamsPositionMap = new HashMap<>();
-        this.kvTablesMap =  new TreeMap<String, InMemoryKVTable>();
```

Comment:
it is more consistent to initialize this in the contructor as all other fields are also initialized in the constructor.

Reply:
also, all other fields are non final. as we set them to null in deleteScope. though logically that doesnt really give us anything. but since that is the pattern being used for all other objects, i would suggest we keep the same pattern for this field too.

Reply:
I don't like the idea of having data members point to null values and wouldn't want to repeat the same just for consistency.

Reply:
we should never be accessing a "deleted" scope object. 
The purpose of setting all fields in the scope object to null should not cause NPE. And if it does, it would point to a bug that should be fixed.

Reply:
Whenever possible, I'd want to avoid null references to class data members in the code, as a good coding practice...